Srcs/heiftojpeg/heiftojpeg.cpp

```diff
@@ -0,0 +1,183 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "hevcimagefilereader.hpp"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Magick++.h>
+#include <list>
+
+using namespace std;
```

Comment:
nitpick - can get rid of all `std::` prefixes below

Reply:
done